packages/BetterPhpDocParser/PhpDoc/ArrayItemNode.php

```diff
@@ -14,30 +14,30 @@ final class ArrayItemNode implements PhpDocTagValueNode, Stringable
     use NodeAttributes;
 
     /**
-     * @param String_::KIND_*|null $kindValueQuoted
+     * @deprecated
      */
+    public int|null $kindValueQuoted = String_::KIND_DOUBLE_QUOTED;
+
+    /**
+     * @deprecated
+     */
+    public int|null $kindKeyQuoted = String_::KIND_DOUBLE_QUOTED;
+
     public function __construct(
         public mixed $value,
-        public mixed $key,
-        public int|null $kindValueQuoted = null,
```

Comment:
dropping this constructor args is also a BC problem

Reply:
I don't think so, if there is any unknown usage of the class, as it's final and I remove only optional parameters and making one optional, those calls will just pass extra parameters to the constructor.
Is there something I missed?